sensor_msgs/PointCloud2 objectCloud
bool recognized
string name
geometry_msgs/PoseStamped[] graspPoses
